reason-react-native/src/types/TV.re

```diff
@@ -0,0 +1,12 @@
+type parallax = {
```

Comment:
For stuff that we pass in to components, our convention is to use bs.obj, not Js.t.